processing/src/main/java/io/druid/query/search/search/SearchQueryConfig.java

```diff
@@ -27,12 +27,32 @@
  */
 public class SearchQueryConfig
 {
+  public static final String CTX_KEY_STRATEGY = "searchStrategy";
+
   @JsonProperty
   @Min(1)
   private int maxSearchLimit = 1000;
 
+  @JsonProperty
+  private String searchStrategy = AutoStrategy.NAME;
```

Comment:
After improving ```auto``` strategy by avoiding computing ```timeFilteredBitmap```, I think we can use it as the default search strategy. It shows nearly optimal performance. Please refer to https://github.com/druid-io/druid/pull/3792#discussion_r95194559.